python/observability-online-eval/livekit/livekit-openai.py

```
@@ -45,12 +53,13 @@ async def web_search(self, query: str) -> str:
         tavily_client = TavilyClient(api_key=TAVILY_API_KEY)
         try:
             response = tavily_client.search(query=query, search_depth="basic")
-            if response.get('answer'):
-                return response['answer']
-            return str(response.get('results', 'No results found.'))
+            if response.get("answer"):
+                return response["answer"]
+            return str(response.get("results", "No results found."))
         except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)